### ISSUE TITLE
Is there a documentation for Java developer?

### ISSUE THREAD

**Issue by MuemmelMann456** [tradingview/lightweight-charts-android#46]
Hello,

I have 0 knowledge about Kotlin and I want to ask whether there is a documentation for Java developer?

With kind regards

MuemmelMann456

**Comment by makedonsky94** [comment]
Hi! Unfortunately there is no any documentation for Java developers. We planning to implement Interoperability with Java and write some documentation about this.

**Comment by MuemmelMann456** [comment]
Hello makedonsky94,
 
Is it now possible to program with the API in Java?
Then I can get help from Stackoverflow.

With kind regards

MuemmelMann456

**Comment by makedonsky94** [comment]
Yes, you can use Kotlin classes and methods from Java. Kotlin code compiles into java bytecode (dex in Android), so there is  full compatibility between Kotlin and Java.
There is a basic documentation about the interop https://kotlinlang.org/docs/java-to-kotlin-interop.html

**Comment by jatinsakshiB** [comment]
There is no doc for realtime candlestick chart in Kotlin

**Comment by makedonsky94** [comment]
You can check the [demo](https://github.com/tradingview/lightweight-charts-android/blob/master/app/src/main/java/com/tradingview/lightweightcharts/example/app/view/charts/RealTimeEmulationFragment.kt) for that case
